src/components/password/password.stories.mdx

```diff
@@ -0,0 +1,209 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs";
+import LinkTo from "@storybook/addon-links/react";
+import StyledSystemProps from "../../../.storybook/utils/styled-system-props";
+import Password from ".";
+import * as stories from "./password.stories.tsx";
+
+<Meta title="Password" parameters={{ info: { disable: true } }} />
+
+# Password
+
+Provide a way for the user to securely enter a password.
+
+## Contents
+
+- [Quick Start](#quick-start)
+- [Examples](#examples)
+- [Props](#props)
+
+## Quick Start
+
+```javascript
+import Password from "carbon-react/lib/components/password";
+```
+
+## Examples
+
+### Default
+
+Please ensure that the `forceObscurity` prop is `true` on form submission, this will ensure browsers do not remember
+the inputs content. We have set the `autocomplete` attribute to `off` on the component also, but this is not widely 
+supported across different browsers and can have unpredictable results.
```

Comment:
1. As we want to highlight this as an important security disclaimer, what do you think about making this paragraph bold?

```suggestion
**Warning - Please ensure that the `forceObscurity` prop is `true` on form submission, this will ensure browsers do not remember the inputs content.** 

We have set the `autocomplete` attribute to `off` on the component also, but this is not widely supported across different browsers and can have unpredictable results.
```